Cargo.lock

```diff
@@ -376,7 +410,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "72394f3339a76daf211e57d4bcb374410f3965dcc606dd0e03738c7888766980"
 dependencies = [
  "anstyle",
- "bitflags",
+ "bitflags 1.3.2",
```

Comment:
dependency duplication, can this be prevented by updating other deps that us older versions?

Reply:
Unfortunately, I won't be able to solve bitflags `1.3`, as there are multiple crates depending on the older version. E.g. https://github.com/image-rs/image-png/pull/400 whom state that it won't be solved soon.